pandas/conftest.py

```diff
@@ -112,6 +112,35 @@ def axis_series(request):
     return request.param
 
 
+@pytest.fixture
```

Comment:
i would rather move this into test_info.py as this overlaps with our generalized fixtures for frames (this is different and only used in 1 place).